index.js

```diff
@@ -16,6 +16,7 @@ class Plex {
     this.plexToken = config.plex_token;
     this.host = config.host || 'localhost';
     this.port = config.port || '32400';
+    this.secure = config.secure || false;
```

Comment:
It would be nicer to use https instead of secure. Https is explicit regarding what the user will turn on with this option.

Reply:
Plex themselves call it "secure" - so behind the scenes it might be https, but it might be better to go with their nomenclature?

Reply:
Fair enough! 👍
Let's go with secure!